public/components/trace_analytics/requests/services_request_handler.ts

```diff
@@ -197,30 +196,6 @@ export const handleServiceMapRequest = async (
     }
   }
 
-  if (currService) {
-    await handleDslRequest(http, DSL, getRelatedServicesQuery(currService), mode, dataSourceMDSId)
```

Comment:
also remove the function to get query if it's no longer used?

Reply:
yes deleted.